e2e-tests/cypress/e2e/self-service/ee01-tabview.cy.ts

```diff
@@ -0,0 +1,318 @@
+import { Common } from '../utils/common';
+
+describe('self-service Login', () => {
+  it('Sign In to self-service', { retries: 2 }, () => {
+    Common.LogintoAAP();
+  });
+});
+
+describe('Execution Environment Tabview Tests', () => {
+  beforeEach(() => {
+    // Navigate to EE tab page and wait for it to load
+    cy.visit('/self-service/ee');
+    cy.wait(2000);
+
+    cy.url({ timeout: 15000 }).should('include', '/self-service/ee');
+    cy.get('main', { timeout: 15000 }).should('be.visible');
+  });
+
+  it('Validates Catalog and Create tabs are visible and switchable', () => {
+    cy.get('body').then($body => {
+      // Catalog tab
+      if ($body.text().includes('Catalog')) {
+        cy.contains('Catalog').should('exist');
+      } else {
+        cy.log('ℹ️ Catalog tab label not found');
+      }
+
+      // Create tab
+      if ($body.text().includes('Create')) {
+        cy.contains('Create').should('exist');
+      } else {
+        cy.log('ℹ️ Create tab label not found');
+      }
+    });
+
+    // Switch to Create tab and verify main remains visible
+    cy.contains('Create').click({ force: true });
+    cy.wait(1000);
+    cy.get('main', { timeout: 15000 }).should('be.visible');
+
+    // Switch back to Catalog tab and verify main remains visible
+    cy.contains('Catalog').click({ force: true });
+    cy.wait(1000);
+    cy.get('main', { timeout: 15000 }).should('be.visible');
+  });
+
+  it('Validates Catalog tab: empty state CTA redirects to Create tab', () => {
+    // Ensure Catalog tab is active (it should be default, but be explicit)
+    cy.get('body').then($body => {
+      if ($body.text().includes('Catalog')) {
+        cy.contains('Catalog').click({ force: true });
+        cy.wait(1000);
+      }
+    });
+
+    cy.get('main', { timeout: 15000 }).should('be.visible');
+
+    cy.get('body').then($body => {
+      const text = $body.text();
+      const hasEmptyState =
+        text.includes('No Execution Environment definition files, yet') ||
```

Comment:
Shouldn't we restrict to the exact text we have added in this page instead of falling back to different variants?